### DISCUSSION TITLE
Include fingerprints in addition to checksums

### DISCUSSION THREAD

**Discussion by rabitt** [mir-dataset-loaders/mirdata#457] [Ideas]
During the presentation of this library, several people requested that we also include audio finger prints in addition to checksums. During validation, if the checksum doesn't match, this allows us to see if the audio matches or not, and could give us information like if there's an offset in time.

Does anyone have an opinion on which fingerprinter to use?

**Comment by keunwoochoi** [comment]
I haven't heard of any method and not sure if there's anything specialized for digital audio. Unless there's a reliable method and its implementation, I'd say we can wait until it becomes a real problem.

**Comment by rabitt** [comment]
The two I've seen most are:
* [AcoustID](https://acoustid.org/) which would play nicely with any dataset that has MusicBrainz IDs
* [echoprint](https://github.com/spotify/echoprint-codegen)
